FinalTask/src/main/java/driver/Driver.java

```diff
@@ -0,0 +1,108 @@
+package driver;
+
+import org.openqa.selenium.Capabilities;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.openqa.selenium.remote.RemoteWebDriver;
+
+import java.net.URL;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.concurrent.TimeUnit;
+
+public class Driver {
+    private static final String DESTINATION_URL = "https://gmail.com";
```

Comment:
It is better to store values for URL, username, access_token in property file, because they can be different for environments